docs/source/tutorials/faq.ipynb

```diff
@@ -611,6 +611,53 @@
     "cl.fit(X, y, find_label_issues_kwargs={\"coarse_search_range\": [0.05, 0.1], \"fine_search_size\": 2})"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### How do I generate cluster IDs and pass them to find_issues for the Underperforming Group Issue Manager?"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The underperforming group issue manager provides the option for passing pre-computed\n",
+    "cluster IDs to `find_issues`. These cluster IDs can be obtained by clustering\n",
+    "the features using algorithms such as K-Means, DBSCAN, HDBSCAN etc. Note that\n",
+    "\n",
+    "* K-Means requires specifying the number of clusters explicitly.\n",
+    "* DBSCAN is sensitive to the choice of `eps` (radius) and `min_samples` (minimum samples for each cluster).\n",
+    "\n",
+    "Below is sample code on how to generate cluster IDs and pass them to `find_issues`: "
```

Comment:
rewrite to not mention issue managers. Also no need to mention pros/cons of different clustering algorithms, just mention you can detect underperforming groups based on any clustering method of your choice (K-means, DBSCAN, hierarchical clustering, etc.). By default, Datalab will detect underperforming groups using the ... clustering algorithm.